docs/reference/slate/change.md

```diff
@@ -50,6 +50,33 @@ function onSomeEvent(event, change) {
 }
 ```
 
+### `withMutations`
+`withMutations(customChange: Function) => Change`
+
+This method calls the provided `customChange` function with the current instance of the `Change` object as the first argument. While `customChange` is executing, normalization is temporarily suppressed, but normalization will be executed once the `customChange` function completes execution.
+
+The purpose of `withMutations` is to allow a sequence of change operations that should not be interrupted by normalization. For example:
+
+```js
+/**
+ * Only allow block nodes in documents.
+ *
+ * @type {Object}
+ */
+validateNode(node) {
+  if (node.object != 'document') return
+  const invalids = node.nodes.filter(n => n.object != 'block')
+  if (!invalids.size) return
+
+  return (change) => {
+    change.withMutations((c) => {
+      invalids.forEach((child) => {
+        change.removeNodeByKey(child.key)
```

Comment:
This should be `c.removeNodeByKey`

Reply:
Sure should, there was a similar mistake in the the other example too